src/_scss/layouts/default/header/nav/_menu.scss

```diff
@@ -22,6 +22,9 @@
             &:first-child {
                 margin-left: 0px;
             }
+            
+            @import "../../../../../components/_comingSoon.scss";
```

Comment:
You can't have `@import` [inside mixins](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#nested-import)